.env

```diff
@@ -1,2 +1,5 @@
 MONGODB_HOST = mongodb+srv://agriplus12345:agriplusxxx@cluster0-jn5sz.mongodb.net/feedback?retryWrites=true&w=majority
 JWT_SECRET = 123
+EMAIL_ID = sakibkhan111296@gmail.com
```

Comment:
don't put your email, password in `.env` file

Reply:
done